Interfaces/API/Dirac.py

```diff
@@ -1008,7 +1008,7 @@ def __printOutput( self, fd = None, message = '' ):
   #     directory = directory[:-1]
   #
   #   if printOutput:
-  #     for fileKey, metaDict in listing['Value']['Successful'][directory]['Files'].items():
+  #     for fileKey, metaDict in listing['Value']['Successful'][directory]['Files'].iteritems():
```

Comment:
I evaluate every time: there are cases in which the move to iteritems() is impossible. So, the changes I made here in this respect are minimal.